### ISSUE TITLE
Wrong usage message.

### ISSUE THREAD

**Issue by prvnkumark** [pan0pt1c0n/knock-knock#1]
Hi,

Usage message and README instructions do not match.
## Usage - ./knockd_eval.py [Input File] [Output File]

Example - ./knockd_eval.py iplist.txt outfile.txt
## Example will test default knockd sequence to identify opened services for all ports on systems listed in input file, and output results in outfile.txt

There is no ./knockd_eval.py and i am unable to use ./knockdefault.py iplist output.txt 2
Option 2 is throwing usage error.

**Comment by Themercee** [comment]
Yes I have the same error. Just don't enter the mode and it will run, the range is hardcoded.
